src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
To the answer the first question, yes, but I don't know of another place that isn't true. An instance of jQuery.Event is a normalized event object with those methods.

As for the second, if all goes well, maybe all we need is a clarification in the docs. Something like, "The native event object is available at `event.originalEvent`, if the original event is not a simulated event (e.g. "focus")."